gen-migration.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
Since this is not a fully automated process - and sometimes doesn't work - I would not commit it here... we can keep a gist somewhere...  It should be something more generic, not just `invenio-vocabularies`.

Reply:
Let's remove it for now since it's an extra. I am not sure about the practical side of the gist. I mean `run-tests.sh` is generic, but it's present in each repo. Let's revisit later: https://github.com/inveniosoftware/invenio-vocabularies/issues/60

Reply:
Yeay, thanks 👼